README.rst

```diff
@@ -68,29 +70,29 @@ Pre-requisites:
 
 * On Windows, please follow the `Comprehensive Installation instructions
   <https://github.com/nexB/scancode-toolkit/wiki/Comprehensive-Installation>`_.
-  Make sure you use Python 2.7 32 bits from
-  https://www.python.org/ftp/python/2.7.15/python-2.7.15.msi
+  Make sure you use Python 3.6.8 32 bits from
+  https://www.python.org/ftp/python/3.6.8/python-3.6.8.exe
 
-* On macOS, install Python 2.7 from
-  https://www.python.org/ftp/python/2.7.15/python-2.7.15-macosx10.6.pkg
+* On macOS, install Python 3.6.8 from
+  https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.6.pkg
 
   Next, download and extract the latest ScanCode release from
   https://github.com/nexB/scancode-toolkit/releases/
 
-* On Linux install the Python 2.7 "devel" and these packages using your
+* On Linux install the Python 3.6 "devel" and these packages using your
   distribution package manager:
 
   * On Ubuntu 14, 16 and 18 use:
-    ``sudo apt-get install python-dev xz-utils zlib1g libxml2-dev libxslt1-dev bzip2``
+    ``sudo apt-get install python3.6-dev xz-utils zlib1g libxml2-dev libxslt1-dev bzip2``
 
   * On Debian and Debian-based distros use:
-    ``sudo apt-get install python-dev xz-utils zlib1g libxml2-dev libxslt1-dev libbz2-1.0``
+    ``sudo apt-get install python3.6-dev xz-utils zlib1g libxml2-dev libxslt1-dev libbz2-1.0``
 
   * On RPM distros use:
-    ``sudo yum install python-devel xz-libs zlib libxml2-devel libxslt-devel bzip2-libs``
+    ``sudo yum install python3.6-devel xz-libs zlib libxml2-devel libxslt-devel bzip2-libs``
```

Comment:
Same as above, are you sure that `python3.6-devel` is the correct package name and is available?

Reply:
Unfourtunately, Yum handles varsions a bit diffrent. Because of that, `python36-devel` does not exist and only Python 3.7 is availible.

(Tested on Fedora in Docker)

Reply:
You rock :bowing_man:  ... but what about EPEL?

Reply:
I think that EPEL also does not have Python 3.6 version of `python-devel`...

Reply:
In case if Python 2.7 is still supported, I can update instructions to only use Python 2.7 on RPM.

But the real solution would be to update ScanCode to support newer versions of Python and then also update instructions to just install any (newer) Python 3 version.

Reply:
>  to just install any (newer) Python 3 version.

Yes, but we will never support anything before Python 3.6

Reply:
But is now Python 3.7 supported?